### DISCUSSION TITLE
Linksys1900acsv2 安装报错

### DISCUSSION THREAD

**Discussion by lihonfu** [messense/aliyundrive-webdav#168] [Q&A] [answered]
### 问题描述

arm_cortex-a9.ipk
arch all 1
arch noarch 1
arch arm_cortex-a9_vfpv3-d16 10
root@OpenWrt:~#
安装报错!

### 重现步骤

wget https://github.com/messense/aliyundrive-webdav/releases/download/v1.0.0/aliyundrive-webdav_1.0.0_arm_cortex-a9.ipk
opkg install aliyundrive-webdav_1.0.0_arm_cortex-a9.ipk

### 版本

1.0.0

### 运行平台

Linksys1900acsv2

### 日志

安装不了,忘了错误提示了..

**Comment by messense** [comment, marked as answer]
`/etc/opkg.conf` 中增加 

```
arch all 1
arch noarch 1
arch arm_cortex-a9_vfpv3-d16 10
arch arm_cortex-a9 100
```

之后再尝试安装。
